.github/workflows/ci.yml

```diff
@@ -23,6 +23,14 @@ jobs:
           cache: pnpm
       - name: Install dependencies
         run: pnpm install --ignore-scripts
```

Comment:
`--ignore-scripts` is added to accelerate the build step

Reply:
Thanks! That works.

I was trying to get it running in the CI since it didn't work locally for some reason.

Reply:
you can try using GitHub workspace. the devcontainer set up the environment https://github.com/ast-grep/langs/tree/main/.devcontainer

Reply:
Oh, with the dev container, I found that it was just a Node.js version issue